libs/island-ui/contentful/src/lib/RichTextRC/defaultRenderNode.tsx

```diff
@@ -169,6 +169,27 @@ export const defaultRenderNodeObject: RenderNode = {
       </Box>
     )
   },
+  [INLINES.EMBEDDED_ENTRY]: (node) => {
+    const amount = node?.data?.target?.fields?.amount
+    if (typeof amount !== 'number') return null
+
+    let postfix = 'krónur'
+
+    const amountString = String(amount)
+
+    if (amountString.endsWith('1') && !amountString.endsWith('11')) {
+      postfix = 'króna'
+    }
+
+    // Format the amount so it displays dots (Example of a displayed value: 2.700 krónur)
+    const formatter = new Intl.NumberFormat('de-DE')
+
+    return (
+      <span>
+        {formatter.format(amount)} {postfix}{' '}
```

Comment:
Consider using a template string for this. I think that's clearer than having the space before and after the postfix
```suggestion
        {`${formatter.format(amount)} ${postfix} `}
```

Reply:
I've removed the space after the posfix since we might want to have a comma or period right after "krónur" so we need to be able to freely choose what the next character is via the CMS rich text content.